### PR TITLE
feat: capability to send the client response to clipboard

### PR DESCRIPTION
**Pull request by juantarrel**
Incredible tool, thank you for creating it, it has been very useful for me. I Created this simple pr to add the ability to save the client's response to the clipboard, it's not a big change, but it's something useful for me.

### CODE REVIEW

**Comment by tbckr** [comment]
Thanks for your contribution! I will have a look into it!

File: cli/root_test.go [hunk 1]
```diff
@@ -104,6 +105,39 @@ func TestRootCmd_SimplePromptOnly(t *testing.T) {
...
+
+	root := newRootCmd(mem.Exit, config, mockIsPipedShell(false, nil), api.MockClient(strings.Clone(expected), nil))
+	root.cmd.SetOut(writer)
+
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		var buf bytes.Buffer
+		_, err := io.Copy(&buf, reader)
+		_ = clipboard.WriteAll(buf.String())
```
Comment (tbckr) [inline cli/root_test.go:126]: Here we are overriding the values which were set in the root command. Therefore we are not actually testing the functionality.

File: cli/root_test.go [hunk 2]
```diff
@@ -104,6 +105,39 @@ func TestRootCmd_SimplePromptOnly(t *testing.T) {
...
+		require.NoError(t, err)
+		require.NoError(t, reader.Close())
+		require.Equal(t, expected, buf.String())
+		require.Equal(t, clipboardBuffer, buf.String())
+	}()
+
+	root.Execute([]string{"--clipboard", prompt})
+	require.Equal(t, 0, mem.code)
+	require.NoError(t, writer.Close())
+
```
Comment (tbckr) [inline cli/root_test.go:137]: Maybe we could do something like this:

```suggestion
textInClipboard := clipboard.ReadAll()
require.Equals(t, expected, textInClipboard)
```

With this we could remove the go func part, io.Pipe and writer.

**Review by tbckr — CHANGES_REQUESTED** [review]
I like your solution! Maybe we could change the test a little bit.

File: cli/root_test.go (hunk 1 above)
Comment (juantarrel) [reply cli/root_test.go]: Sorry you are right

Comment (juantarrel) [reply cli/root_test.go]: fixed

File: cli/root_test.go (hunk 2 above)
Comment (juantarrel) [reply cli/root_test.go]: fixed

**Review by tbckr — APPROVED** [review]

**Comment by tbckr** [comment]
The error in the CI pipeline is expected. The Github runner does not have a clipping tool installed. I will fix this test error after the merge.
Thanks for your contribution!

**Pull request merged**